modules/programs/zed-editor.nix

```diff
@@ -0,0 +1,97 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.zed-editor;
+  jsonFormat = pkgs.formats.json { };
+
+  configDir = if pkgs.stdenv.hostPlatform.isDarwin then
+    "Library/Application Support/zed" # TODO: see if this is actually the correct directory
```

Comment:
Many apps support using XDG dirs even on Mac. I'm pretty sure Zed does since I have a `.config/zed` immediately after downloading and running it. Can this just use the xdg dirs regardless of platform and allow an override if needed?

Reply:
I am waiting for the fix for this to be merged, but I see no activity from the mantainer. Can someone else take over or are we just waiting?

Reply:
I can confirm that config directory is in `~/.config/zed` both on linux and mac